coursemapper-kg/concept-map/src/main.py

```
@@ -4,13 +4,48 @@
 
 from cm_worker import Worker
+import json
+import hashlib
 
 
 # Create the worker
 worker = Worker(Config.REDIS_HOST, Config.REDIS_PORT, Config.REDIS_DB, Config.REDIS_PASSWORD)
 worker.log_to_console = True
 
 
+def create_job_and_set_state(pipeline_name, args_dict, state="pending"):
```

Ruff (0.12.2)

Missing return type annotation for public function `create_job_and_set_state`

Add return type annotation: `None`

(ANN201)